src/node.hpp

```diff
@@ -189,6 +189,11 @@ class Node {
   // https://stackoverflow.com/a/776523/467327
   static size_t SORotate(size_t n, uint32_t c);
 
+  // Output single node to string.
```

Comment:
There's a specific sense that this is a node to string, and I wouldn't know that by looking at the header file. Perhaps `NodeIdAndLeavesToString`? And the whole tree version could be `NodeIdAndLeavesToStringForTopology`?

I usually think of a Newick string when I think of a generic "tree to string" thing.